yamls/distributed_ppo_test.yaml

```diff
@@ -21,4 +21,141 @@ command: |-
   composer test_single_controller_ppo.py --file_path /mnt/config/parameters.yaml
 
 parameters:
-  pretrain_model_name: Qwen/Qwen2.5-3B-Instruct
+  variables:
+    max_seq_len: 6000
```

Comment:
Can you change the value here to 10420 and `max_gen_len` to 8192. Not entirely needed since we could change it ourselves, but I think the default should be the ideal values